CONTRIBUTING.md

```diff
@@ -81,40 +82,67 @@ There are five steps to submit pull request:
 
 ### Create a new branch
 
-```
-$ git branch fix-issue-123
-$ git checkout fix-issue-123
+```bash
+git branch fix-issue-123
+git checkout fix-issue-123
 ```
 
 ### Make your changes
 
 Make sure your code is following [TypeScript Style Guide](https://google.github.io/styleguide/tsguide.html).
 
 Lint your code:
-```
-$ pnpm lint
+
+```bash
+pnpm lint
 ```
 
 ### Test your changes
 
 You should add a new test file for your changes into [test](/tree/main/test) folder, the file should has extension `.test.ts`.
 
 Run tests:
-```
-$ pnpm test
+
+```bash
+pnpm test
 ```
 
 Generate coverage report:
+
+```bash
+pnpm coverage
 ```
-$ pnpm coverage
-```
+
+### Test changes in a real project
+
+To test the changes you made, you can use the [`playground`](./playground)
+project.
+
+1. In the windicss root project directory, run:
+
+    ```bash
+    pnpm install
+    ```
+
+2. In the first terminal, run `pnpm run dev` in the windicss project directory, and in the
+   second one open the playground directory and run `pnpm run dev`.
```

Comment:
```suggestion
2. In the first terminal, run `pnpm dev` in the windicss project directory, and in the
   second one open the playground directory and run `pnpm dev`.
```
In pnpm you can run scripts without `run`. just to keep everything consistent in this document.